frontend/src/pages/api/nodes.js

```diff
@@ -2,15 +2,23 @@ import { ExplorerApi } from "../../libraries/explorer-wamp";
 
 export default async function (req, res) {
   try {
-    await new ExplorerApi(req).call("node-telemetry", [
+    let ip_address =
+      req.headers["x-forwarded-for"] ||
+      req.connection.remoteAddress ||
+      req.socket.remoteAddress ||
+      req.connection.socket.remoteAddress;
+
+    res.send("node telemetry received.");
+
+    return await new ExplorerApi(req).call("node-telemetry", [
```

Comment:
I don't see a point in returning anything. HTTP 200 with blank response should be enough.